shared/appengine.go

```diff
@@ -25,18 +27,29 @@ import (
 // runtimeIdentity contains the identity of the current AppEngine service when
 // running on GAE, or empty when running locally.
 var runtimeIdentity struct {
-	LocationID  string
-	AppID       string
-	Service     string
-	Version     string
+	LocationID string
+	AppID      string
+	Service    string
+	Version    string
+
+	// Internal details of the application identity
 	application *apps.Application
 }
 
+// clients contains heavyweight (e.g. with connection pools) clients that
+// should be bound to the runtime instead of each request in order to be
+// reused. They are initialized and authenticated at startup using the
+// background context; each request should use its own context.
+var clients struct {
```

Comment:
`clients` here seems to refer multiple clients but it is in fact just one. How about just `client`?

Reply:
You're right correct; there's going to be more coming soon :)